src/metatrain/utils/scaler.py

```diff
@@ -0,0 +1,237 @@
+from typing import Dict, List, Union
+
+import metatensor.torch
+import numpy as np
+import torch
+from metatensor.torch import TensorMap
+from metatensor.torch.atomistic import ModelOutput
+
+from .additive import remove_additive
+from .data import Dataset, DatasetInfo, TargetInfo, get_all_targets
+from .jsonschema import validate
+
+
+class Scaler(torch.nn.Module):
+    """
+    A class that scales the targets of regression problems to unit standard
+    deviation.
```

Comment:
So this is not centering at the same time? It might be worth to explain why. I guess this is because the other additive models are expected to bring the mean to zero already?

Reply:
Yes, the "mean" is already removed by the composition model

Reply:
Added a comment